wpilibNewCommands/src/main/native/cpp/frc2/command/PrintCommand.cpp

```diff
@@ -8,9 +8,9 @@
 
 using namespace frc2;
 
-PrintCommand::PrintCommand(const wpi::Twine& message)
-    : CommandHelper{[str = message.str()] { wpi::outs() << str << "\n"; }, {}} {
-}
+PrintCommand::PrintCommand(std::string_view message)
+    : CommandHelper{
+          [str = std::string(message)] { wpi::outs() << str << "\n"; }, {}} {}
```

Comment:
Should we make wpi::outs() support std::string_view?

Reply:
It's addressing a lifetime issue.  This is a capture that can run much later, and string_view could be pointing to a temporary.